engineering/architecture.md

```diff
@@ -30,6 +30,7 @@ These documents are notes about a specific feature written to get feedback befor
 
 - [Import Previewing](/engineering/architecture/import-preview-api)
 - [Syncing Database Objects](/engineering/architecture/syncing)
+- [Numbers](/engineering/architecture/numbers)
```

Comment:
Could we name this the same thing as the page ("Number display and entry") for clarity?

Reply:
Fixed in d0a1ef4fcb7ae6b2690d80ebdf2631aadb58e008